### PR TITLE
AArch64 native build support

### PR DESCRIPTION
**Pull request by PhatFree**
Code changes to allow for aarch64 native build

### CODE REVIEW

**Review by darkwood101 — APPROVED** [review]

**Comment by edhay** [comment]
Reviewed by @edhay

File: common/scripts/build-linux-sbsa.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ build_sbsa_kernel_driver()
     echo $arch
     if [[ $arch = "aarch64" ]]
```
Comment (chetan-rathore) [inline common/scripts/build-linux-sbsa.sh:42]: @PhatFree:  Can we have line 42 changed to below as done in build-grub.sh
  if [ $arch == "aarch64" ]; then

File: common/scripts/build-uefi.sh [hunk 2]
```diff
@@ -67,14 +71,15 @@ fi
...
+        PATH="$PATH:$CROSS_COMPILE_DIR"
+    fi
     source ./edksetup.sh
     make -C BaseTools
     export EDK2_TOOLCHAIN=$UEFI_TOOLCHAIN
-    arch=$(uname -m)
-    if [[ $arch = "aarch64" ]]
-    then
+
+    if [[ $arch = "aarch64" ]]; then
```
Comment (chetan-rathore) [inline common/scripts/build-uefi.sh:82]: Can we have same syntax followed.
ex:   if [ $arch == "aarch64" ]

**Comment by PhatFree** [comment]
Hi Chetan, even though this script is only verified on bash, single equals has better compatibility across POSIX shells. double brackets is not as portable, but is generally preferred. 

I've updated the grub build script for consistency.

**Pull request merged**